MericCelik/assignment1/LinkedList.java

```diff
@@ -0,0 +1,43 @@
+
+public class LinkedList{
+    LinkedListNode head;
+
+    public LinkedList(){
+        this.head = null;
+    }
+
+    public void add(LinkedListNode newnode){
+        LinkedListNode temp = this.head;
+        while(temp.next!=null){
+            temp = temp.next;
+        }
+        temp.next = newnode;
+    }
+
+    public void add(Object newobject){
+        LinkedListNode newnode = new LinkedListNode(newobject);
+        LinkedListNode temp = this.head;
+        while(temp.next!=null){
+            temp = temp.next;
+        }
+        temp.next = newnode;
```

Comment:
optional: Lines 19-23 are a copy of lines 10-14. Perhaps you could replace 19-23 with add(newnode)?

Reply:
Great idea!